### ISSUE TITLE
Manipulate is Changing value automatically

### ISSUE THREAD

**Issue by aburousan** [WolframResearch/WolframLanguageForJupyter#128]
In Wolframe Engine 13.0, Manipulate is automatically changing it's
![Screenshot from 2022-03-17 10-03-50](https://user-images.githubusercontent.com/60939745/158738176-3e1c7a8d-392d-463c-a0ce-3c2426e753d7.png)
 value.
![Screenshot from 2022-03-17 10-03-53](https://user-images.githubusercontent.com/60939745/158738181-070bad09-4fd4-490f-afe5-c314b3c21b81.png)
System: Ubuntu 20.04.4 LTS

**Comment by AnonymoZ** [comment]
That's true. **Manipulate[...]** and **DynamicModule[...]** are currently not fully supported by Jupyter as well as **Animate[...]**. You can produce animations although not control them. 
You can try alternatives like Bokeh, MPLD3 and IPyVolume; the latter two are found on GitHub itself.
